exwm-workspace.el

```diff
@@ -252,26 +281,106 @@ The optional FORCE option is for internal use only."
       (xcb:+request exwm--connection
           (make-instance 'xcb:ChangeWindowAttributes
                          :window window-id :value-mask xcb:CW:EventMask
-                         :event-mask xcb:EventMask:SubstructureRedirect))))
-  (xcb:flush exwm--connection)
-  ;; We have to delay making the frame visible until the
-  ;; override-redirect flag has been set.
-  (select-frame-set-input-focus (car exwm-workspace--list))
-  (dolist (i exwm-workspace--list)
-    (set-frame-parameter i 'visibility t)
-    (lower-frame i)
-    (set-frame-parameter i 'fullscreen 'fullboth))
-  (raise-frame (car exwm-workspace--list))
+                         :event-mask xcb:EventMask:SubstructureRedirect)))
+    (xcb:flush exwm--connection)
+    ;; We have to delay making the frame visible until the
+    ;; override-redirect flag has been set.
+    (set-frame-parameter frame 'visibility t)
+    (lower-frame frame)
```

Comment:
Not my decision to make, but I think you should, and call `exwm--select-frame` instead. That would make our idea of which frame we're on consistent, I think.